Gemfile.lock

```diff
@@ -15,74 +15,74 @@ GEM
     RubyInline (3.14.0)
```

Comment:
Let's avoid making an updates to the Gemfile in this PR

Reply:
I'm struggling with reverting the gemfile back to before updating it 
- can I update it manually with the correct ruby-in-line version?

Reply:
Is it in one commit? If so you can revert that one commit. Otherwise you could make another commit to change it back to the current version